articles/azure/asz-sco

```diff
@@ -0,0 +1,262 @@
+
+- Azure blob storage (Block)
+Block blobs (generically known as object storage) is optimized for streaming and storing cloud objects, and are a good choice for storing documents, media files, backups, and etc.
+
+- Azure blob storage (Apend)
+Append blobs are optimized for append operations. An append blob can be updated only by adding a new block to the end. Append blobs are a good choice for scenarios such as logging, where new data needs to be written only to the end of the blob.
+
+- Azure Queue Storage
+In designing applications for scale, application components are often decoupled, so that they can scale independently. Queue storage provides a reliable messaging solution for asynchronous communication between application components, whether they're running in the cloud, on the desktop, on an on-premises server, or on a mobile device. Queue storage also supports managing asynchronous tasks and building process workflows. A storage account can contain any number of queues, and a queue can contain any number of messages, up to the capacity limit of the storage account. Individual messages may be up to 64 KB in size.
+
+- Azure Table Storage
+Table storage is Microsoft's NoSQL key/attribute store - it has a schemaless design, making it different from traditional relational databases. With a schemaless data store, it's easy to adapt your data as the needs of your application evolve. Table storage is easy to use, so developers can create applications quickly.
+Table storage is a key-attribute store, which means that every value in a table is stored with a typed property name. The property name can be used for filtering and specifying selection criteria. A collection of properties and their values comprise an entity. Since table storage is schemaless, two entities in the same table can contain different collections of properties, and those properties can be of different types.
+You can use table storage to store flexible datasets, such as user data for web applications, address books, device information, and any other type of metadata that your service requires. For today's Internet-based applications, NoSQL databases like table storage offer a popular alternative to traditional relational databases.
+A storage account can contain any number of tables, and a table can contain any number of entities, up to the capacity limit of the storage account.
+
+> [!NOTE]
+> All storage in Azure Stack is taken from a common pool. When designing your service for UKCloud for Microsoft Azure, consider the best usage of your storage. For example, if you have a large object storage requirement, consider using the UKCloud's Cloud storage platform as part of your solution. 
+ 
+## Service Management
+- API 
```

Comment:
Add more detail here? Something like:

Enables you to programmatically interact with UKCloud for Microsoft Azure resources